src/translations/en.json

```
@@ -2100,6 +2100,13 @@
               "yaml": "YAML file",
               "storage": "UI controlled"
             },
+            "dialog_new": {
+              "header": "Add dashboard",
+              "create_empty": "New dashboard from scratch",
+              "create_empty_description": "Start with an empty dashboard from scratch",
+              "default": "Overview dashboard",
+              "default_description": "Display all the devices of your home"
```

Should we be more specific here too. Maybe add grouped by area? 

```suggestion
              "default_description": "Display your devices grouped by area"
```

We should consider that this strategy is shown next to other strategies.